### DISCUSSION TITLE
Mod not running on server

### DISCUSSION THREAD

**Discussion by KuschelKaktus88** [P3pp3rF1y/SophisticatedBackpacks#1058] [Q&A]
So i am playing on aternos server and whenever i install the mod plus the core mod to it it says i need a mod, called modern keybinding. but the problem here is that the mkb mod is only client side and cant be installed on aternos servers. so my question is there any way to make a version of this mod that doesnt require modern keybinding??

**Comment by P3pp3rF1y** [comment]
You are probably running a fabric version of the mod so I can't support you on that (you would want to go here https://github.com/Salandora/SophisticatedBackpacks/issues 

But I have also seen multiple issues where Aternos was the actual cause of the issue so you may want to consider hosting your server somewhere else.
